docs/api-docs/catalog/price-list-overview.md

```diff
@@ -103,12 +103,33 @@ lineNumbers: true
 }]
 ```
 
+
+## Price list assignments
+
+The price list assignment assigns a price list to a specific sales channel. This association lets you define custom pricing for shoppers only on a specific external site or platform. 

```

Comment:
I would be somewhat cautious here- It will work for channels (storefronts), but it won't affect pricing in different omnichannel environments (Facebook, Amazon, Instagram, eBay, etc.)

Reply:
I have updated the scope of Price List assignments